diff --git a/base.ts b/base.ts
index 7bfa008..ba89421 100644
--- a/base.ts
+++ b/base.ts
@@ -16,7 +16,7 @@ import { Configuration } from "./configuration";
 // @ts-ignore
 import globalAxios, { AxiosPromise, AxiosInstance } from "axios";

-export const BASE_PATH = "http://localhost".replace(/\/+$/, "");
+export const BASE_PATH = (window.location.protocol+'//'+window.location.host).replace(/\/+$/, "");

 /**
  *
