diff --git a/api.ts b/api.ts
index 6b9b3c7..22c443d 100644
--- a/api.ts
+++ b/api.ts
@@ -414,19 +414,19 @@ export interface ProductCategoryjsonld {
    * @type {string}
    * @memberof ProductCategoryjsonld
    */
-  context: string;
+  '@context': string;
   /**
    *
    * @type {string}
    * @memberof ProductCategoryjsonld
    */
-  id: string;
+  '@id': string;
   /**
    *
    * @type {string}
    * @memberof ProductCategoryjsonld
    */
-  type: string;
+  '@type': string;
   /**
    *
    * @type {string}
@@ -488,19 +488,19 @@ export interface ProductCollectionjsonld {
    * @type {string}
    * @memberof ProductCollectionjsonld
    */
-  context: string;
+  '@context': string;
   /**
    *
    * @type {string}
    * @memberof ProductCollectionjsonld
    */
-  id: string;
+  '@id': string;
   /**
    *
    * @type {string}
    * @memberof ProductCollectionjsonld
    */
-  type: string;
+  '@type': string;
   /**
    *
    * @type {string}
@@ -562,19 +562,19 @@ export interface ProductLocationjsonld {
    * @type {string}
    * @memberof ProductLocationjsonld
    */
-  context: string;
+  '@context': string;
   /**
    *
    * @type {string}
    * @memberof ProductLocationjsonld
    */
-  id: string;
+  '@id': string;
   /**
    *
    * @type {string}
    * @memberof ProductLocationjsonld
    */
-  type: string;
+  '@type': string;
   /**
    *
    * @type {string}
@@ -655,19 +655,19 @@ export interface ProductStockjsonld {
    * @type {string}
    * @memberof ProductStockjsonld
    */
-  context: string;
+  '@context': string;
   /**
    *
    * @type {string}
    * @memberof ProductStockjsonld
    */
-  id: string;
+  '@id': string;
   /**
    *
    * @type {string}
    * @memberof ProductStockjsonld
    */
-  type: string;
+  '@type': string;
   /**
    *
    * @type {string}
@@ -766,19 +766,19 @@ export interface Productjsonld {
    * @type {string}
    * @memberof Productjsonld
    */
-  context: string;
+  '@context': string;
   /**
    *
    * @type {string}
    * @memberof Productjsonld
    */
-  id: string;
+  '@id': string;
   /**
    *
    * @type {string}
    * @memberof Productjsonld
    */
-  type: string;
+  '@type': string;
   /**
    *
    * @type {string}
